### DISCUSSION TITLE
Building the WebApp

### DISCUSSION THREAD

**Discussion by martin8310** [tbnobody/OpenDTU#311] [General]
Hallo zusammen, 

ich bin begeistert von dem Projekt. Super Arbeit!!!

Die Anleitung habe ich soweit verstanden, aber ich verstehe einen Punkt nicht "Building the WebApp" was soll man da erstellen und wo und wozu? Wie führe ich das aus, gehe ich dazu unter Windows einfach in das Verzeichnis und tippe da "yarn install" ein? Oder tippe ich das irgendwo im VS Code ein? oder ....?

Gruß
Martin

**Comment by ghost** [comment]
"Building the WebApp" ist (zunächst) ohne Bedeutung. Einfach ab hier
https://github.com/tbnobody/OpenDTU#using-the-pre-compiled-bin-files --> Flash with Espressif Flash Download Tool (Windows) --> "First configuration" vorgehen.
